docs/local.rst

```diff
@@ -188,6 +188,28 @@ The default settings module loaded by ``edx-platform`` is ``tutor.production``.
 
 Of course, your settings should be compatible with the docker install. You can get some inspiration from the ``production.py`` settings modules generated by Tutor, or just import it as a base by adding ``from .production import *`` at the top of ``mysettings.py``.
 
+
+Saving logs to files instead of sending them to Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default tracking logs and app logs are sent to docker, and are visible with ``tutor local logs``. Another option is to persist logs in a directory alongside the database data.
+To direct logs from an openedx component (lms, cms or workers) edit the docker-compose file at ``$(tutor config printroot)/env/local/docker-compose.yml`` and add the followinf line to the lms/cms and worker containers, replaceing SERVICE_NAME with the relevant service name (lms, cms, lms-worker, cms-worker):
+
+    volumes:
+      - ../apps/openedx/settings/lms/:/openedx/edx-platform/lms/envs/tutor/
+      - ../apps/openedx/settings/cms/:/openedx/edx-platform/cms/envs/tutor/
+      ...
+      - ../../data/logs/SERVICE_NAME:/openedx/logs # <-- Add this line
+
+If you also want logs from nginx to be saved to disk instead of sent to docker, add the following to the nginx container in ``$(tutor config printroot)/env/local/docker-compose.yml``:
+
+    volumes:
+      - ../apps/nginx:/etc/nginx/conf.d/
+      - ../../data/openedx:/var/www/openedx:ro
+      ...
+      - ../../data/logs/lms:/var/log/nginx # <-- Add this line
```

Comment:
I'd rather not encourage users to manually modify rendered templates. This is fragile, as most users won't realise that files are overwritten every time we run `tutor config save`.

I propose to store all logs in `/openedx/data/logs`. This is the current configuration for the `LOG_DIR` setting in the lms/cms. Thus, logs will be persisted on disk by default.